.github/workflows/ci-reverse-lab.yml

```diff
@@ -0,0 +1,60 @@
+# Copyright (c) 2023 SolarWinds, LLC.
+# All rights reserved.
+
+name: CI Reverse Lab Scan
+
+on:
+  workflow_dispatch:
+
+jobs:
+  # act -j ci-reverse-lab-scan --container-architecture linux/arm64
+  ci-reverse-lab-scan:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Set up Ruby 3.1 and bundle
+        uses: ruby/setup-ruby@v1
+        with:
+          ruby-version: 3.1
+
+      - name: Build apm-ruby
+        id: build-apm-ruby-ci-reverse-lab
+        run: |
+          sudo apt-get update && sudo apt-get install -y --no-install-recommends ruby ruby-dev g++ make swig bison
+          sudo gem install bundler
+          sudo echo 'gem: --no-document' >> ~/.gemrc
+          sudo bundle install --without development --without test
+          sudo bundle exec rake fetch_oboe_file["prod"]
+          sudo gem build solarwinds_apm.gemspec
+          CURRENT_GEM=$(ls | grep solarwinds_apm-*.gem)
+          GEM_VERSION=$(echo "$CURRENT_GEM" | grep -oP '(?<=solarwinds_apm-)[0-9]+\.[0-9]+\.[0-9]+(\.[a-z0-9]+)?(?=\.gem)')
+          echo "current_gem=$CURRENT_GEM" >> $GITHUB_OUTPUT
+          echo "gem_version=$GEM_VERSION" >> $GITHUB_OUTPUT
+
+      - name: Print Gem and Version (in case some debugging is needed)
+        run: |
+          echo "Gem name:       ${{ steps.build-apm-ruby-ci-reverse-lab.outputs.current_gem }}"
+          echo "Gem version:       ${{ steps.build-apm-ruby-ci-reverse-lab.outputs.gem_version }}"
+
+      - name: Run Reverse Lab Scan
+        run: |
+          docker run --rm \
+            -u $(id -u):$(id -g) \
+            -v "$(pwd)/:/packages:ro" \
+            -v "$(pwd)/reports:/reports" \
+            -e RLPORTAL_ACCESS_TOKEN=${{ secrets.REVERSE_LAB_TOKEN }} \
+            reversinglabs/rl-scanner-cloud \
+            rl-scan \
+              --rl-portal-server solarwinds \
+              --rl-portal-org SolarWinds \
+              --rl-portal-group "SaaS  - Agents- SWO" \
```

Comment:
Thanks for the correct group name. 

No way, the correct portal group is double spaced. 
I was trying all day last week with `SaaS - Agents- SWO` and got 
```
Started: Scanning version
Info: Not found.
```

Reply:
I feel your pain :grin: